### PR TITLE
Add Senuti.app v1.3.3

### PR DESCRIPTION
**Pull request by laeryn**
Senuti is an application that allows you to transfer music and videos from your iPhone or iPod to iTunes.

### CODE REVIEW

File: Casks/senuti.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'senuti' do
+  version '1.3.3'
+  sha256 '23bf22a7c468753ab71a6464d920f05223a093ec17bc9e3a7d3be8ea6913474f'
+
+  url "https://www.fadingred.com/files/senuti/senuti_#{version}.zip"
+  name 'Senuti'
+  homepage 'https://www.fadingred.com/senuti/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/senuti.rb]: `:commercial`.

**Comment by laeryn** [comment]
Thanks, fixed it.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by laeryn** [comment]
Ok, I squashed 'em. I will remember to squash next time.

Thanks!

**Pull request merged**